### PR TITLE
MINOR: Add 3.7.1 to system tests

### PR DESCRIPTION
**Pull request by soarez**
Part of the release process for 3.7.1.

### CODE REVIEW

File: tests/docker/Dockerfile [hunk 1]
```diff
@@ -99,7 +99,7 @@ RUN mkdir -p "/opt/kafka-3.3.2" && chmod a+rw /opt/kafka-3.3.2 && curl -s "$KAFK
 RUN mkdir -p "/opt/kafka-3.4.1" && chmod a+rw /opt/kafka-3.4.1 && curl -s "$KAFKA_MIRROR/kafka_2.12-3.4.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-3.4.1"
 RUN mkdir -p "/opt/kafka-3.5.2" && chmod a+rw /opt/kafka-3.5.2 && curl -s "$KAFKA_MIRROR/kafka_2.12-3.5.2.tgz" | tar xz --strip-components=1 -C "/opt/kafka-3.5.2"
 RUN mkdir -p "/opt/kafka-3.6.2" && chmod a+rw /opt/kafka-3.6.2 && curl -s "$KAFKA_MIRROR/kafka_2.12-3.6.2.tgz" | tar xz --strip-components=1 -C "/opt/kafka-3.6.2"
-RUN mkdir -p "/opt/kafka-3.7.0" && chmod a+rw /opt/kafka-3.7.0 && curl -s "$KAFKA_MIRROR/kafka_2.12-3.7.0.tgz" | tar xz --strip-components=1 -C "/opt/kafka-3.7.0"
+RUN mkdir -p "/opt/kafka-3.7.1" && chmod a+rw /opt/kafka-3.7.1 && curl -s "$KAFKA_MIRROR/kafka_2.12-3.7.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-3.7.1"
```
Comment (chia7712) [inline tests/docker/Dockerfile:102]: Is the repo available?

Comment (chia7712) [reply tests/docker/Dockerfile:102]: https://s3-us-west-2.amazonaws.com/kafka-packages/kafka_2.12-3.7.1.tgz does not work for me :(

**Review by chia7712 — COMMENTED** [review]
@soarez thanks for this patch!

File: tests/docker/Dockerfile (hunk 1 above)
Comment (soarez) [reply tests/docker/Dockerfile:102]: Thanks for having a look @chia7712. The upload to S3 has been done now. Could you try that link again?

Comment (chia7712) [reply tests/docker/Dockerfile:102]: for another: 3.7.1 is nonexistent in https://repo.maven.apache.org/maven2/org/apache/kafka/kafka-streams/

Comment (soarez) [reply tests/docker/Dockerfile:102]: Thanks for pointing this out too. Looks like I had skipped the step to release the repos in nexus. These artifacts are these now.

**Review by chia7712 — APPROVED** [review]
LGTM

**Pull request merged**